plutus-use-cases/src/Plutus/Contracts.hs

```diff
@@ -40,34 +40,3 @@ These two steps depend on each other because the transaction fee is a
 function of the size of the transaction including its
 inputs.
 -}
-
-{- Note [Validity Interval's upper bound]
```

Comment:
Do we want to remove this note before the cardano-ledger bug is resolve? ADR-13 gives a clearer semantic for validity range but it doesn't invalidate this comment, does it?